ecosystem/node-checker/configuration_examples/ait3_validator.yaml

```diff
@@ -40,9 +40,7 @@ evaluator_args:
   tps_args:
     emit_args:
       mempool_backlog: 25000
-      wait_millis: 2
-      burst: false
-      do_not_check_stats_at_end: true
+      target_tps: 0
```

Comment:
can we just remove those target_tps: 0?

Reply:
@banool needs to recheck NHC changes, and can apply this too.